src/commands/wizard/lwcGenerationCommand.ts

```diff
@@ -22,9 +35,81 @@ export class LwcGenerationCommand {
                             panel.dispose();
                             return resolve();
                         }
+                    },
+                    {
+                        type: 'getQuickActionStatus',
+                        action: async (_panel, data, callback) => {
+                            // TODO: Hook this up to function that parses landing_page.json.
+                            const sobjects = [
+                                'Account',
+                                'Contact',
+                                'Opportunity',
+                                'SomeOther'
+                            ];
+                            if (callback) {
+                                const quickActionStatus =
+                                    await LwcGenerationCommand.checkForExistingQuickActions(
+                                        sobjects
+                                    );
+                                callback(quickActionStatus);
+                            }
+                        }
                     }
                 ]
             );
         });
     }
+
+    static async checkForExistingQuickActions(
+        sobjects: string[]
+    ): Promise<SObjectQuickActionStatus> {
+        return new Promise<SObjectQuickActionStatus>(async (resolve) => {
+            const results: SObjectQuickActionStatus = { sobjects: {} };
+            results.sobjects = {};
+
+            sobjects.forEach((sobject) => {
+                const quickActionStatus: QuickActionStatus = {
+                    view: false,
+                    edit: false,
+                    create: false
+                };
+                quickActionStatus.view =
+                    LwcGenerationCommand.checkForExistingQuickAction(
+                        sobject,
+                        'view'
+                    );
+                quickActionStatus.edit =
+                    LwcGenerationCommand.checkForExistingQuickAction(
+                        sobject,
+                        'edit'
+                    );
+                quickActionStatus.create =
+                    LwcGenerationCommand.checkForExistingQuickAction(
+                        sobject,
+                        'create'
+                    );
+
+                results.sobjects[sobject] = quickActionStatus;
+            });
+
+            return resolve(results);
+        });
+    }
+
+    private static checkForExistingQuickAction(
+        sobject: string,
+        qaName: string
+    ): boolean {
+        const expectedDirName = `${sobject}.${qaName}.quickAction-meta.xml`;
```

Comment:
Nit: this variable name could probably be slightly more appropriate, e.g. `expectedMetadataFilename` or somesuch.